Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -39,6 +39,9 @@ public class ErrorPlugin : EventProcessorPluginBase {
 
             error.Data[Error.KnownDataKeys.TargetInfo] = targetInfo;
 
+            if (context.StackSignatureData.Count > 0)
```

Comment:
If I remove this line, then the stack signature will be different. I want the signature to be based on the manual stacking key only. I think it makes sense to check just the known key instead. I'll update the code.

I'll write some tests. Thanks!